include/benchmark/benchmark.h

```diff
@@ -337,6 +337,15 @@ BENCHMARK_EXPORT size_t
 RunSpecifiedBenchmarks(BenchmarkReporter* display_reporter,
                        BenchmarkReporter* file_reporter, std::string spec);
 
+// TimeUnit is passed to a benchmark in order to specify the order of magnitude
+// for the measured time.
+enum TimeUnit { kNanosecond, kMicrosecond, kMillisecond, kSecond };
+
+BENCHMARK_EXPORT TimeUnit GetDefaultTimeUnit();
+
+// Sets the default time unit the benchmarks use
```

Comment:
add a comment documenting that it only works if called prior to the main benchmark loop please.